src/main/java/com/github/maxopoly/environmentaleffects/datarepresentations/MobConfig.java

```diff
@@ -142,8 +145,10 @@ public MobConfig(String identifier, EntityType type, String name,
 
 	public Location findSpawningLocation(Location loc) {
```

Comment:
It would be very helpful if @Maxopoly  or @BlackXnt could write an english-language explanation of what this algorithm is attempting to do. I have some concerns in general, but I'm not sure what it's supposed to do, so I'm having trouble formulating an understanding sufficient to my concerns. Just a short logic/process explanation would suffice.